src/main/kotlin/no/nav/familie/oppdrag/service/OppdragServiceImpl.kt

```diff
@@ -22,13 +23,25 @@ class OppdragServiceImpl(
 
     @Transactional(rollbackFor = [Throwable::class])
     override fun opprettOppdrag(utbetalingsoppdrag: Utbetalingsoppdrag, oppdrag: Oppdrag, versjon: Int) {
+        if (oppdragErAlleredeSendt(utbetalingsoppdrag, oppdrag, versjon)) {
```

Comment:
Føler dette blev mye endringer. I 99% av alle tilfeller så kommer oppdraget ikke være sendt. Sånn att en sjekk om det allerede er sendt er unødvendig. 
Hvorfor ikke bare kalle på opprettOppdrag og ta en catch på duplicate exception?

Reply:
...Ikke helt enig returen blir av typen db-feil constraint og teoretisk så kan det være en annen constraint som slår til og i tillegg er spørringen mot db ikke så dyr ..

Reply:
Ja teoretiskt hadde den kunnat ge `DuplicateKeyException` om det ær conflict på constraint också.
Vi har inga andra constaints, och om man vill vara sæker på det så har man en integrasjonstest som lagrar 2 exakt lika uppdrag, med 2 olika behandlingId.

I utgångspunkten tycker jag det ær onødigt att kalla databasen varje gång før något som hænder en enstaka gång. Operationen ær billig, men jag førsøker alltid att unngå att kalla databasen om jag inte behøver

Reply:
... Vi kan diskutere det på mandag ..

Reply:
God poeng! 
Hadde man kunnet gjort noe slikt:
```kotlin
try {
            oppdragLagerRepository.opprettOppdrag(oppdragLager, versjon)
        } catch (e: DuplicateKeyException) {
            sjekkOmOppdragFinnes()
```
Blir ju litt styggt kanskje, men samtidig så unngår man den sjekken for hvert oppdrag :) 
Det har ikke veldig mye å si, men tenker att for den dagen dere skal gjøre satsendring, så tar det ekstra oppslaget mot databasen en del ekstra tid :) Og er ju unødvendig i mange tilfeller

Reply:
Japp, jeg kan endre det.